### DISCUSSION TITLE
Starting exe

### DISCUSSION THREAD

**Discussion by Cobenick** [audiamus/AaxAudioConverter#66] [Q&A] [answered]
The file downloaded just fine but I can't get it to open. Help?

**Comment by Cobenick** [comment, marked as answer]
I am running Windows 10, Thanks!

**Reply by audiamus** [reply]
Does your system allow to install software from anywhere, including packages not digitally signed? Look for any system messages where you may have to confirm installation.

Did you re-download the package as your first download may have failed nonetheless? 

(You could also verify the integrity of your download against its MD5 hash: https://github.com/audiamus/AaxAudioConverter/blob/master/res/Setup.dat )

**Reply by Cobenick** [reply]
I ran the trouble shooter and got it to work. Thanks for your help! I look forward to using this!
